xgcm/grid.py

```diff
@@ -1834,7 +1824,7 @@ def cumint(self, da, axis, **kwargs):
         da_i : xarray.DataArray
             The cumulatively integrated data
         """
-
+        # first check if axis is present in da
```

Comment:
This seems a bit confusing over here?

Reply:
Yeah, I forget to delete that, it was supposed to just remind me to do that step 😅